WalletWasabi/Blockchain/Analysis/FeesEstimation/AllFeeEstimate.cs

```diff
@@ -65,6 +67,80 @@ public AllFeeEstimate(AllFeeEstimate other, bool isAccurate)
 	[JsonProperty]
 	public Dictionary<int, int> Estimations { get; }
 
+	/// <summary>
+	/// Estimations where we try to fill out gaps for all valid time spans.
+	/// </summary>
+	public IEnumerable<(TimeSpan timeSpan, FeeRate feeRate)> WildEstimations
+	{
+		get
+		{
+			var timeSpan = TimeSpan.FromMinutes(20);
+			IEnumerable<(TimeSpan timeSpan, FeeRate feeRate)> convertedEstimations = Estimations.Select(x => (TimeSpan.FromMinutes(x.Key * 10), new FeeRate((decimal)x.Value)));
+			if (!convertedEstimations.Any())
+			{
+				return Enumerable.Empty<(TimeSpan timeSpan, FeeRate feeRate)>();
+			}
+
+			var wildEstimations = new List<(TimeSpan timeSpan, FeeRate feeRate)>();
+			var prevFeeRate = FeeRate.Zero;
+			while (timeSpan <= TimeSpan.FromDays(7))
+			{
+				var smaller = convertedEstimations.LastOrDefault(x => x.timeSpan <= timeSpan);
+				var bigger = convertedEstimations.FirstOrDefault(x => x.timeSpan >= timeSpan);
+				if (smaller == default)
+				{
+					smaller = bigger;
+				}
+				if (bigger == default)
+				{
+					bigger = smaller;
+				}
+
+				var diffTime = bigger.timeSpan - smaller.timeSpan;
+				var diffFeeRateSatPerByte = bigger.feeRate.SatoshiPerByte - smaller.feeRate.SatoshiPerByte;
+				var distance = timeSpan - smaller.timeSpan;
+				decimal ratio;
+				if (diffTime == distance || diffTime == TimeSpan.Zero)
+				{
+					ratio = 1;
+				}
+				else
+				{
+					ratio = (decimal)(distance / diffTime);
+				}
+
+				var feeRateSatPerByte = smaller.feeRate.SatoshiPerByte + (ratio * diffFeeRateSatPerByte);
+
+				FeeRate feeRate = new FeeRate(feeRateSatPerByte);
```

Comment:
To simplify new expression:

```suggestion
				FeeRate feeRate = new(feeRateSatPerByte);
```